src/views/FavoriteFoodMain.js

```diff
@@ -0,0 +1,134 @@
+import React from 'react';
+import FavoriteFoods from './FavoriteFoods';
+import jsonData from '../assets/data/data.json';
+import CardColumns from 'react-bootstrap/CardColumns';
+
+
+class FavoriteFoodMain extends React.Component {
+
+  constructor(props){
+    super(props);
+    this.state = {
+      gallery: jsonData
+    }
+  }
+  
+
+
+
+
+  render() {
+    return(
+      <div>
+      <CardColumns>
+
+      <FavoriteFoods
+      src = "https://picsum.photos/id/1041/200/300"
+      title = "Wave"
+      description = "A wave and a cloudy sky."
+       / >
```

Comment:
I assume your next step is to generate this list from `this.state.gallery`?

Most commonly in React you'll see attributes indented and no space around the `=`:

```js
      <FavoriteFoods
        src="https://picsum.photos/id/1041/200/300"
        title="Wave"
        description="A wave and a cloudy sky."
        />
```